react/src/pages/analysis/Analysis.tsx

```diff
@@ -107,11 +107,16 @@ const analyses = [
     createAnalysis(8, 'AN20032', '2020-06-20 7:07 AM', 'User C', '3839', ['CC773', 'CC774', 'CC775'], 'CRE', '22hrs', PipelineStatus.ERROR),
 ];
 
+type ParamTypes = {
+    analysisID: string | undefined
+  }
+
 export default function Analysis() {
     const classes = useStyles();
-    const [detail, setDetail] = useState(false);
+    const { analysisID }= useParams<ParamTypes>();
+    const [detail, setDetail] = useState(true);
     const [cancel, setCancel] = useState(false);
-    const [activeRow, setActiveRow] = useState<AnalysisRun | null>(null);
+    const [activeRow, setActiveRow] = useState<AnalysisRun | null>(analyses.filter(analysis => analysis.analysisID === analysisID)[0]);
```

Comment:
You can just use [`Array.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead of `Array.filter()[0]`